packages/suite/src/views/dashboard/components/AssetsCard/components/AssetGrid/index.tsx

```diff
@@ -0,0 +1,144 @@
+import React from 'react';
+import styled, { useTheme } from 'styled-components';
+import { Network } from '@wallet-types';
+import { CoinLogo, Icon, variables } from '@trezor/components';
+import { FiatValue, Ticker, Translation } from '@suite-components';
+import { CoinBalance } from '@wallet-components';
+import { isTestnet } from '@wallet-utils/accountUtils';
+import * as routerActions from '@suite-actions/routerActions';
+import { useActions, useAccountSearch } from '@suite-hooks';
+
+const Col = styled.div`
+    display: flex;
+    align-items: center;
+    padding: 16px 0px;
+    color: ${props => props.theme.TYPE_DARK_GREY};
+    font-size: ${variables.FONT_SIZE.NORMAL};
+    font-weight: ${variables.FONT_WEIGHT.MEDIUM};
+    padding-right: 20px;
+`;
+
+const CoinNameWrapper = styled.div`
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    font-weight: ${variables.FONT_WEIGHT.DEMI_BOLD};
+    cursor: pointer;
+`;
+
+const UpperRowWrapper = styled.div`
+    display: flex;
+    justify-content: space-between;
+    width: 302px;
+    padding: 0px 15px 15px 15px;
+    border-bottom: 1px solid ${props => props.theme.STROKE_GREY};
+`;
+
+const Coin = styled.div`
+    white-space: nowrap;
+    text-overflow: ellipsis;
+    overflow: hidden;
+    align-items: center;
+`;
+
+const LogoWrapper = styled.div`
+    padding-right: 12px;
+    display: flex;
+    align-items: center;
+`;
+
+const CoinGridWrapper = styled.div`
+    width: 330px;
+    height: 118px;
+    margin: 0px 9px 18px 0px;
+    padding: 19px 16px 18px 12px;
+    border-radius: 6px;
+    background-color: #ffffff;
+`;
+
+const CryptoBalanceWrapper = styled(Col)`
+    display: flex;
+    align-items: center;
+    justify-content: center;
+`;
+
+const FailedCol = styled(Col)`
+    color: ${props => props.theme.TYPE_RED};
+    font-size: ${variables.FONT_SIZE.SMALL};
+    font-weight: ${variables.FONT_WEIGHT.REGULAR};
+
+    @media screen and (max-width: ${variables.SCREEN_SIZE.SM}) {
+        grid-column: 1 / 3;
+        margin-left: 25px;
+        border-bottom: 1px solid ${props => props.theme.STROKE_GREY};
+    }
+`;
+
+const FiatBalanceWrapper = styled.span`
+    color: ${props => props.theme.TYPE_LIGHT_GREY};
+    margin-left: 0.5ch;
+`;
+
+interface Props {
+    network: Network;
+    failed: boolean;
+    cryptoValue: string;
+}
+
+const AssetGrid = React.memo(
+    ({ network, failed, cryptoValue }: Props): JSX.Element => {
```

Comment:
Return type (`JSX.Element`) is optional here and can be removed as TypeScript should infer it.

Reply:
The problem came from old develop branch and inspiration assets coponent.